certs/cert.go

```diff
@@ -37,7 +37,7 @@ func DecryptCertificate(cert []byte, privateKey []byte, password []byte) (*Certi
 	block, _ := pem.Decode(privateKey)
 
 	var decrypted []byte
-	if decrypted, err = x509.DecryptPEMBlock(block, password); err != nil {
+	if decrypted, err = x509.DecryptPEMBlock(block, password); err != nil { //nolint:staticcheck
```

Comment:
`x509.DecryptPEMBlock` is deprecated. I'm just suppressing the linter error for now.

Reply:
Cool

Reply:
Add this as a comment in the code too, so that we'll know the reason in the future